src/status_im/ui/screens/routing/back_actions.cljs

```diff
@@ -0,0 +1,29 @@
+(ns status-im.ui.screens.routing.back-actions)
+
+(def back-actions {:chat                              :default
```

Comment:
shouldn't you use `:intro-wizard/navigate-back` for some screens here?

Reply:
no for now, this PR is only to prevent broken states of the app, later we could decide what to do with cancel actions

Reply:
but what does it mean for this action? it prevents hardware button and 3 finger slide back but if you press the cancel button it works?

Reply:
yes

Reply:
Hardware back button is not prevented here, it still works.

Reply:
I think there's no way to disable hw back button unless resorting to RN's `BackHandler`.